iothub/service/src/FileNotification.cs

```diff
@@ -7,16 +7,19 @@ namespace Microsoft.Azure.Devices
     using Newtonsoft.Json;
 
     /// <summary>
-    /// Contains properties for file upload notifications 
+    /// Contains properties for file upload notifications
     /// </summary>
     public class FileNotification
     {
         [JsonProperty(PropertyName = "deviceId", DefaultValueHandling = DefaultValueHandling.Ignore)]
         public string DeviceId { get; set; }
 
         [JsonProperty(PropertyName = "blobUri", DefaultValueHandling = DefaultValueHandling.Ignore)]
+#pragma warning disable CA1056 // Uri properties should not be strings, changing this would break the API
```

Comment:
nit: how do you feel about suppressing this via an attribute instead:
`[System.Diagnostics.CodeAnalysis.SuppressMessage("Design", "CA1056:Uri properties should not be strings", Justification = "changing this would break the API")]`

1 line of code vs 2!

Reply:
I like this MUCH better.